### PR TITLE
Move schemas to appendix

### PR DESCRIPTION
**Pull request by decentralgabe**
Schema references in each section.

### CODE REVIEW

**Comment by decentralgabe** [comment]
Also opened https://github.com/decentralized-identity/spec-up/issues/38

File: package.json [hunk 1]
```diff
@@ -9,13 +9,15 @@
...
   },
   "dependencies": {
+    "markdown-it": "^9.1.0",
     "spec-up": "^0.10.3"
   },
   "devDependencies": {
     "ajv": "^8.10.0",
     "ajv-formats": "^2.1.1",
-    "mocha": "^9.2.2",
-    "del-cli": "^4.0.1"
```
Comment (JaceHensley) [inline package.json:18]: What are these dep changes for?

**Review by JaceHensley — APPROVED** [review]
LGTM, just a question

File: package.json (hunk 1 above)
Comment (decentralgabe) [reply package.json:18]: npm i broke without these 😕

**Pull request merged**